src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -904,3 +904,180 @@ def acquireTiledArea(streams, stage, area, overlap=0.2, settings_obs=None, log_p
     executeAsyncTask(future, task.run)
 
     return future
+
+
+def acquireOverview(streams, stage, area, focus, ccd, overlap=0.2, settings_obs=None, log_path=None, zlevels=None,
+                    registrar=REGISTER_GLOBAL_SHIFT, weaver=WEAVER_MEAN, focusing_method=FocusingMethod.NONE):
+    """
+    Start a tiled acquisition task for the given streams (SEM or FM) in order to
+    build a complete view of the TEM grid. Needed tiles are first acquired for
+    each stream, then the complete view is created by stitching the tiles.
+
+    Parameters are the same as for TiledAcquisitionTask
+    :return: (ProgressiveFuture) an object that represents the task, allow to
+        know how much time before it is over and to cancel it. It also permits
+        to receive the result of the task, which is a list of model.DataArray:
+        the stitched acquired tiles data
+    """
+    # Create a progressive future with running sub future
+    future = model.ProgressiveFuture()
+    future.running_subf = model.InstantaneousFuture()
+    future._task_lock = threading.Lock()
+
+    task = AcquireOverviewTask(streams, stage, area, focus, ccd, future, overlap, settings_obs,
+                               log_path, zlevels,
+                               registrar, weaver, focusing_method)
+    future.task_canceller = task.cancel  # let the future cancel the task
+
+    future.set_progress(end=task.estimate_time() + time.time())
+    # connect the future to the task and run in a thread
+    executeAsyncTask(future, task.run)
+
+    return future
+
+
+class AcquireOverviewTask(object):
+    """
+    Copied from class ZStackAcquisitionTask(object):
+    """
+
+    def __init__(self, streams, stage, area, focus, ccd, future, overlap=0.2, settings_obs=None, log_path=None,
+                 zlevels=None,
+                 registrar=REGISTER_GLOBAL_SHIFT, weaver=WEAVER_MEAN, focusing_method=FocusingMethod.NONE):
+        # site and feature means the same
+        self._stage = stage
+        self._future = future
+        self.streams = streams
+        self.area = area
+        self._rois = [area, area]  # list of roi # TODO ask GUI to provide a list of rois
+        self._ccd = ccd  # TODO to delete after ccd.data components is used from streams in do_autofocus_roi
+        self._focus = focus
+        self.focus_rng = self._focus.getMetadata().get(model.MD_POS_ACTIVE_RANGE, None)
+
+        if self.focus_rng is None:
+            # METEOR is running
+            self.focus_rng = self._focus.getMetadata().get(model.MD_SAFE_REL_RANGE, None)
+            if self.focus_rng is not None:
+                self.focus_rng = (self.focus_rng[0] + self._focus.position.value,
+                                  self.focus_rng[1] + self._focus.position.value)
```

Comment:
`self._focus.position.value` -> `self._focus.position.value["x"]`
(and same for y)